test/lib/slots.test.ts

```diff
@@ -0,0 +1,56 @@
+import getSlots from '@lib/slots';
+import {it, expect} from '@jest/globals';
+import MockDate from 'mockdate';
+import dayjs, {Dayjs} from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+MockDate.set('2021-06-20T11:59:59Z');
```

Comment:
Codacy has a fix for the issue: [Replace `'2021-06-20T11:59:59Z'` with `"2021-06-20T11:59:59Z"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
MockDate.set("2021-06-20T11:59:59Z");
```